src/manage-acc.php

```diff
@@ -0,0 +1,191 @@
+<?php
+include_once str_replace('/', DIRECTORY_SEPARATOR, __DIR__ . '/includes/classes/file-utils.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/Path.php');
+include_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/page-head-utils.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/user.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/session-handler.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/page-router.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/db-config.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/db-connector.php');
+
+$_SESSION['organization'] = "jpia";
+$_SESSION['user'] = "user";
+
+$user = new User(1, 'Admin', 'jpia', 'Doe', 'John', 'Michael', 'Jr.', '12', 'A', 'john.doe@example.com', 'Active', 'Voted');
```

Comment:
Notice this may conflict with the User class removed organization property from this class since there is a session for holding an organization session.